es5.js

```diff
@@ -230,6 +255,10 @@ module.exports = {
     'no-useless-call': 2,
     // disallow unnecessary concatenation of literals or template literals
     'no-useless-concat': 2,
+    // disallow unnecessary escape usage
+    'no-useless-escape': 2,
+    // disallow redundant return statements
+    'no-useless-return': 2,
```

Comment:
Hmm, could be problematic. However I'd say both rules are useful, and if someone enters this situation they can always disable whichever rule doesn't fit their usecase with `// eslint-disable-line ...`